guide/sections/part2/global-services.adoc

```diff
@@ -151,7 +151,8 @@ In WIS2 Global Caches provide access to WMO core data for data consumers. This a
 * A Global Cache shall provide the metric defined in this Guide at an HTTP endpoint
 * A Global Cache should make sure that data is downloaded in parallel and downloads are not blocking each other
 
-* The metric ``wmo_wis2_gc_dataserver_status_flag`` will reflect the status of the connection to the download endpoint of the Centre. Its' value will be 1 when the endpoint is up and 0 otherwise.
+* The metric ``wmo_wis2_gc_dataserver_status_flag`` will reflect the status of the connection to the download endpoint of the Centre. Its value will be 1 when the endpoint is up and 0 otherwise.
```

Comment:
```suggestion
* The metric ``wmo_wis2_gc_dataserver_status_flag`` will reflect the status of the connection to the download endpoint of the centre. Its value will be 1 when the endpoint is up and 0 otherwise.
```